source/system/native/MutexUnix.ooc

```diff
@@ -19,7 +19,17 @@ version(unix || apple) {
 			super()
 		}
 		lock: override func {
-			pthread_mutex_lock(this _backend&)
+			version(debugDeadlock) {
+				timeLimitMilliseconds := 1000 //Only whole seconds
```

Comment:
Yeah, and the case where where `tv_nsec > 1 seconds`
```ooc
if (deadline tv_nsec > 1_000_000_000) {
    deadline tv_nsec -= 1_000_000_000
    deadline tv_sec += 1
}
```
Just feels like it is an unnecessary feature that adds some additional rows :)